ui/src/services/experiment/ExperimentStatus.js

```diff
@@ -19,19 +19,19 @@ export const getExperimentStatus = (experiment) => {
     },
     Completed: {
       label: "Completed",
-      color: "#017D73",
+      color: "#00BFB3",
```

Comment:
Can we replace the `color` values here with the `healthColor` values (and replace the property name `healthColor` with `color` in the app)? The current `color` values are only used in the Eui badge from what I see and they should be able to accept primary / success / ...

Reply:
Similar issue here as the above - it seems like `EuiHealth` also uses a darker shade (kind of like in the older versions of the theme) of all the colours instead of the more exciting ones we've been seeing for the headers/titles. I should checked this more thoroughly before I guess 👀 In any case, I've replaced the `color` values with the `healthColor` values (`subdued`/`success`/`warning`/`primary`) and removed the `healthColor` values completely. Any component that used to consume `healthColor` values now consume the new (reverted) `color` values.
![Screenshot 2022-09-13 at 10 55 49 AM](https://user-images.githubusercontent.com/36802364/189797488-5847288e-820a-4942-9209-b1b976eaa534.png)